src/newLaunch/stages/stage8.html

```diff
@@ -1,63 +1,91 @@
 <template>
-  <div class="page animated-page stageContainer stage8 au-animate ">
+  <div class="page animated-page stageContainer stage8 au-animate">
     <div class="infoContainer">
-      <div class="heading heading2 title">Thank you!</div>
+      <div class="heading heading2 title">Thank you! here!</div>
       <div class="subtext">
         <p>Congratulations!</p>
 
         <p>You have created a ${wizardState.launchTypeTitle} according to your specifications.</p>
 
-        <p>You may inspect the ${wizardState.launchTypeTitle} configuration, stored in the JSON format in IPFS, here:
-          <a class='wrappedLink' href="${ipfsURL}" target="_blank" rel="noopener noreferrer">${ipfsURL}</a>.
+        <p>
+          You may inspect the ${wizardState.launchTypeTitle} configuration, stored in the JSON format in IPFS, here:
+          <a class="wrappedLink" href="${ipfsURL}" target="_blank" rel="noopener noreferrer">${ipfsURL}</a>.
           <span class="importantText">Please save this URL.</span>
         </p>
 
-        <p>PrimeDAO Governance needs to approve and initialize your new ${wizardState.launchTypeTitle} before it can
-          appear in and be used in the Prime Launch interface. The approval takes place in the <a
-            href.bind="multiSigWalletUri" target="_blank" rel="noopener noreferrer">Prime Launch multisig wallet</a>.
-          Your ${wizardState.launchTypeTitle} may take up to 5 minutes to appear in the multisig. Reviews generally take
-          5-7 days."</p>
+        <p if.to-view="dataToken">
+          <span class="importantText"
+            >Note that when you generate a new token, the value will only be displayed once, so be sure to write it
+            down</span
+          >
+          <span class="dataTokenWrapper">${dataToken}</span>
```

Comment:
I left a comment about html wrap formatter from vscode.
Can you check if that's the case for you?

Ideally, not major formatting should happen

Reply:
that's the correct setting, no ?
![image](https://user-images.githubusercontent.com/85691163/189075823-5110d08c-2ccb-4ef8-a216-cca03c59470c.png)

Reply:
yeah, then I don't know why you have all those html attribute changes?